### DISCUSSION TITLE
database.ts not found

### DISCUSSION THREAD

**Discussion by maswebber** [adonisjs/core#2802] [Help] [answered]
Hello, i am new to this framework, are ts files for auth, database, and other extra modules in config we added manually or its automatically added to config folder when installing the modules ?
Thanks

**Comment by RomainLanz** [comment, marked as answer]
Hey @maswebber! 👋 

Those files are added when needed, so only when you install the related module.

**Reply by maswebber** [reply]
Hello i just install the module like lucid orm, but the config doesn't appear

**Reply by RomainLanz** [reply]
Have you correctly followed the steps?

They are 3 steps you need to do to install a module, all described in the documentation.
📚 https://docs.adonisjs.com/guides/database/introduction

**Reply by maswebber** [reply]
Thanks, so already install but i didn't see the configure my bad 😅
